R/saveout.R

```diff
@@ -142,18 +142,21 @@ saveout.net <- function(dat, s, out = NULL) {
     if (dat$control$save.nwstats == TRUE) {
       out$stats$nwstats <- list(dat$stats$nwstats)
     }
-    if (dat$control$save.transmat == TRUE) {
-      if (!is.null(dat$stats$transmat)) {
-        row.names(dat$stats$transmat) <- 1:nrow(dat$stats$transmat)
-        out$stats$transmat <- list(dat$stats$transmat)
-      } else {
-        out$stats$transmat <- list(data.frame())
+
+    if (dat$control$tergmLite != TRUE) {
+      if (dat$control$save.transmat == TRUE) {
```

Comment:
There should not be a `save.transmat` parameter at all anymore. tergmLite replaces that